Docs: updates Grafana installation for Red Hat, Fedora, RHEL, SUSE, and OpenSUSE (#65232)
grafana/grafana: docs/sources/setup-grafana/installation/rpm/index.md

aliases:
  - ../../installation/installation/rpm/
  - ../../installation/rpm/

> description: Install guide for Grafana on Red Hat, RHEL, or Fedora.

Author: ```suggestion
description: Install guide for Grafana on Red Hat, RHEL, and Fedora.
```

title: Install Grafana on Red Hat, RHEL, or Fedora
menuTitle: Redhat, RHEL, or Fedora
weight: 400
---

# Install Grafana on Red Hat, RHEL, or Fedora

This topic explains how to install Grafana dependencies, install Grafana on Redhat, RHEL, or Fedora, and start the Grafana server on your system.

You can install Grafana using a YUM repository, using RPM, or by downloading a binary `.tar.gz` file.

> If you install via the `.tar.gz` file, then you must manually update Grafana for each new version.

Reviewer: ```suggestion
If you install via RPM or the `.tar.gz` file, then you must manually update Grafana for each new version.
```

## Install Grafana using YUM

> You can install Grafana from a YUM repository or manually using YUM.

Reviewer: Let's split this up so that the repository option is standalone.  the manual yum install is more closely related to the manual rpm install than to using the repo, and has the same upgrade limitation as rpm or tar.gz install

Author: Done.

> ### Install from YUM repository

Author: ```suggestion
### Install Grafana from the YUM repository
```

| Grafana Enterprise | grafana-enterprise | `https://rpm.grafana.com` |
| Grafana OSS        | grafana            | `https://rpm.grafana.com` |

> **Note:** Grafana Enterprise is the recommended and default edition. It is available for free and includes all the features of the OSS edition. You can also upgrade to the [full Enterprise feature set](https://grafana.com/products/enterprise/?utm_source=grafana-install-page), which has support for [Enterprise plugins](https://grafana.com/grafana/plugins/?enterprise=1&utcm_source=grafana-install-page).

To install Grafana using a YUM repository, complete the following steps:

> 1. Add a new file to your YUM repo using the method of your choice.

Author: ```suggestion
1. Add a file to your YUM repository using the method of your choice.
```

   The following example uses `nano` to add a file to the YUM repo.

   ```bash
   sudo nano /etc/yum.repos.d/grafana.repo
   ```

   ```bash
   [grafana]
   name=grafana
   baseurl=https://rpm.grafana.com
   repo_gpgcheck=1
   enabled=1
   gpgcheck=1
   gpgkey=https://rpm.grafana.com/gpg.key
   sslverify=1
   sslcacert=/etc/pki/tls/certs/ca-bundle.crt
   ```

1. To prevent beta versions from being installed, add the following exclude line to your `.repo` file.

   ```bash
   exclude=*beta*
   ```

1. To install Grafana OSS, run the following command:

   ```bash
   sudo yum install grafana
   ```

1. To install Grafana Enterprise, run the following command:

   ```bash
   sudo yum install grafana-enterprise
   ```

### Install Grafana manually using YUM

> If you install Grafana manually using YUM, then you must manually update Grafana for each new version. The following steps enable automatic updates for your Grafana installation.

Author: ```suggestion
If you install Grafana manually using YUM, you must manually update Grafana for each new version. The following steps enable automatic updates for your Grafana installation.
```

1. Navigate to the [Grafana download page](https://grafana.com/grafana/download).
1. Select the Grafana version you want to install.
   - The most recent Grafana version is selected by default.
   - The **Version** field displays only tagged releases. If you want to install a nightly build, click **Nightly Builds** and then select a version.
1. Select an **Edition**.
   - **Enterprise:** This is the recommended version. It is functionally identical to the open source version, but includes features you can unlock with a license, if you so choose.
   - **Open Source:** This version is functionally identical to the Enterprise version, but you will need to download the Enterprise version if you want Enterprise features.
1. Depending on which system you are running, click the **Linux** or **ARM** tab on the download page.
1. Copy and paste the code from the installation page into your command line and run.

> ## Install Grafana using RPM

Reviewer: yeah we could combine this into a "manually install the rpm package" section, the only difference is the actual command you use to install the rpm package once you've downloaded it (yum vs rpm)

   - **Enterprise** - Recommended download. Functionally identical to the open source version, but includes features you can unlock with a license if you so choose.
   - **Open Source** - Functionally identical to the Enterprise version, but you will need to download the Enterprise version if you want Enterprise features.
1. Depending on which system you are running, click **Linux** or **ARM**.
1. Copy and paste the RPM package URL and the local RPM package information from the installation page into the pattern shown below, then run the commands.

>    ```bash

Author: I _think_ we don't need this but wanted to check with you.
